extra/puppet_sign.rb

```diff
@@ -26,14 +26,14 @@ def log(message)
 SETTINGS         = YAML.load_file(settings_file)
 PUPPETCA         = YAML.load_file(SETTINGS[:settings_directory] + '/puppetca.yml')
 unless PUPPETCA[:enabled]
-  log('PuppetCa is not enabled!')
+  log('PuppetCA smart-proxy module is not enabled!')
   exit 1
 end
 protocol         = PUPPETCA[:enabled] == true ? 'https' : PUPPETCA[:enabled]
 port             = protocol == 'https' ? SETTINGS[:https_port] : SETTINGS[:http_port]
 fqdn             = Socket.gethostbyname(Socket.gethostname).first
-# e.g. https://hostname.localdomain.com:8443/puppet/ca/autosign
-uri              = URI.parse("#{protocol}://#{fqdn}:#{port}/puppet/ca/autosign")
+# e.g. POST https://hostname.localdomain.com:8443/puppet/ca/validate
+uri              = URI.parse("#{protocol}://#{fqdn}:#{port}/puppet/ca/validate")
```

Comment:
This is quite a change I can hardly comment but this looks like some `extra` script.